BccMembers.Api.Client/IBccMembersApiClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using BccMembers.Api.Client.Contracts;
+
+namespace BccMembers.Api.Client
+{
+    public interface IBccMembersApiClient
+    {
+        Task<BccPerson> GetPersonAsync(int id);
```

Comment:
I think we should use personId as paramenter name to be clear

Reply:
👍